package.json

```diff
@@ -5,30 +5,33 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "cypress": "cypress open",
-    "cypress:headless": "cypress run"
+    "cypress": "cypress open --config-file=cypress.config.js",
+    "cypress:run": "cypress run --config-file=cypress.config.js"
   },
   "dependencies": {
-    "@headlessui/react": "^1.4.2",
-    "@vercel/analytics": "^1.1.3",
-    "axios": "^0.24.0",
+    "@headlessui/react": "^1.7.14",
+    "@vercel/analytics": "^1.3.1",
+    "axios": "^1.5.0",
     "ms": "^2.1.3",
     "next": "latest",
-    "qs": "^6.10.3",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-hot-toast": "^2.1.1",
+    "qs": "^6.11.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-hot-toast": "^2.4.0",
     "react-use-form-state": "^0.13.2"
```

Comment:
Can you just update this version rather than using a resolution to resolve the conflict?

Reply:
Looks like there isn't a later version, it was last updated 4 years ago and is stuck on react 16 - probably should get rid of it at some point.